apps/web/src/utils/experienceUtils.ts

```diff
@@ -320,10 +320,30 @@ export const isWorkExperience = (e: AnyExperience): e is WorkExperience =>
   e.__typename === "WorkExperience";
 
 export const compareByDate = (e1: ExperienceForDate, e2: ExperienceForDate) => {
-  const e1EndDate = e1.endDate ? new Date(e1.endDate).getTime() : null;
-  const e2EndDate = e2.endDate ? new Date(e2.endDate).getTime() : null;
-  const e1StartDate = e1.startDate ? new Date(e1.startDate).getTime() : -1;
-  const e2StartDate = e2.startDate ? new Date(e2.startDate).getTime() : -1;
+  // fit AwardExperience to startDate - endDate format
+  const e1Adjusted = e1;
+  const e2Adjusted = e2;
+  if (e1.__typename === "AwardExperience") {
+    e1Adjusted.startDate = e1.awardedDate;
+    e1Adjusted.endDate = e1.awardedDate;
+  }
+  if (e2.__typename === "AwardExperience") {
+    e2Adjusted.startDate = e2.awardedDate;
+    e2Adjusted.endDate = e2.awardedDate;
+  }
```

Comment:
Since `AwardExperience` only has a single date would it make sense to choose either `startDate` or `endDate` instead of setting them both to the same date? 🤔

Reply:
If you only pick say a start date, and end date is null it gets treated as an ongoing experience wouldn't it?
If that is preferable then I can change that, but it would bubble award experiences all to the top then. 

I assigned both start and end to have them sort in the same way as the other experiences.

Reply:
Okay, that makes sense to me 👍